src/client/components/Google_login_button.components/GoogleLoginButton.stories.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import GoogleLoginButton from './GoogleLoginButton';
+
+export default { title: 'GoogleLoginButton' };
+
```

Comment:
it's recommended to add the component name after the title : component: GoogleLoginButton

Reply:
Yes, I forgot it here :)